config/routes.rb

```diff
@@ -1,3 +1,21 @@
 Rails.application.routes.draw do
+  root 'pages#home'
+  get 'addDoctor', to: 'pages#home'
+  get 'appointments', to: 'pages#home'
+  get 'DetailsPage/:id', to: 'pages#home'
+  get 'doctors', to: 'pages#home'
```

Comment:
As @nidalaa mentioned there are routes that might not be necessary. For instance these Rails routes where you are referencing to #home controller. We are using Rails here, remember, to generate JSON responses and not HTML pages (one of the confusions that could have been resolved by separating your Rails API from your React repository)

Reply:
We removed unneeded rails routes.  they were not even being used in our project. thanks.